.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,15 @@ resolves #1235
 ### This pullrequest changes
 
 <!-- Please describe what your pullrequest is changing -->
+
+<cut/>
+
+### Pull Request Readiness Checklist
+
+- [ ] I agree to contribute to the project under OpenCV ([BSD](https://github.com/opencv/opencv/blob/master/LICENSE)) license
+- [ ] The PR is proposed to proper branch ([details](https://github.com/opencv/opencv/wiki/How_to_contribute#before-you-start-contributing-you-should))
+- [ ] There is reference to original bug report and related work
+- [ ] There is test, if code has been changed
+- [ ] There is PR with opencv_extra changes, if any
```

Comment:
=> "If the patch includes a new test(s) that needs new test data, there is a pull request to http://github.com/opencv/opencv_extra from the branch with the same name, that adds the data"